parser/SqlParser.ypp

```diff
@@ -676,7 +686,7 @@ sql_statement:
   | quit_statement {
     $$ = $1;
   }
-  | select_statement {
+  | set_operation_statement {
```

Comment:
That will leads to a conflict -- the parser cannot decide whether a `SELECT` query should be reduced to a select statement or a set operation statement.